examples/demo/.env

```diff
@@ -1 +1,2 @@
 OTELCOL_IMG=omnition/opentelemetry-collector-contrib:latest
+OTELCOL_ARGS=
```

Comment:
What is the purpose of this empty var?

Reply:
Good q, I should add an example to the demo readme (will do in a separate PR). I've been using the docker-compose to check metrics and zpages, but I don't want to break the example with the published image. So during development, I can use `OTELCOL_IMG=otelcol:latest OTELCOL_ARGS="--new-metrics=true" docker-compose up -d` that argument is not supported by the published version.